### PR TITLE
Creation Routes and methods show, index for books

### PR DESCRIPTION
**Pull request by AleFalcon**
## Summary
* Creation routes show and indes for book
* Creation methods for book

## Trello Card
https://trello.com/c/Afa2PT4W/21-crear-rutas-index-y-show-para-book

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
```
Comment (wleoguty91) [inline app/models/rent.rb]: This class should not be here :thinking: see what happend with git

File: app/controllers/book_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class BookController < ApiController
+  include Wor::Paginate
+
+  def index
+    render_paginated Book, each_serializer: BookSerializer
+  end
+
+  def show
+    id = show_params
+    if id.to_i.!= 0
```
Comment (wleoguty91) [inline app/controllers/book_controller.rb]: When you use `find`, it's not necessary add this validations. This is require when you use something like `where`

File: app/serializers/book_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class BookSerializer < ActiveModel::Serializer
+  attribute :id
```
Comment (wleoguty91) [inline app/serializers/book_serializer.rb]: You can put all together `attributes :id, :gender, :author`

File: spec/factories/user.rb [hunk 4]
```diff
@@ -1,11 +1,7 @@
 FactoryBot.define do
   factory :user do
     email { Faker::Internet.email }
-<<<<<<< HEAD
```
Comment (wleoguty91) [inline spec/factories/user.rb]: Be carefull with this

File: spec/models/rent_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (wleoguty91) [inline spec/models/rent_spec.rb]: This file should not be here

File: spec/factories/user.rb (hunk 4 above)
Comment (AleFalcon) [reply spec/factories/user.rb]: Ready

File: app/serializers/book_serializer.rb (hunk 3 above)
Comment (AleFalcon) [reply app/serializers/book_serializer.rb]: Ready

File: app/controllers/book_controller.rb (hunk 2 above)
Comment (AleFalcon) [reply app/controllers/book_controller.rb]: Ready

File: app/models/rent.rb (hunk 1 above)
Comment (AleFalcon) [reply app/models/rent.rb]: where should it be? It was always like that

Comment (nicozar95) [reply app/models/rent.rb]: What @wleoguty91 is trying to said is that this class belongs to another PR

Comment (AleFalcon) [reply app/models/rent.rb]: There modify the dependence of the branch, this was badly put

Comment (AleFalcon) [reply app/models/rent.rb]: There modify the dependence of the branch, this was badly put

File: config/routes.rb [hunk 6]
```diff
@@ -2,5 +2,9 @@
 
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
+  
+  resources :book, only: [:index, :show] do
+    resources :products, only: [:index, :show]
```
Comment (wleoguty91) [inline config/routes.rb]: This resources it's not necesary

Comment (AleFalcon) [reply config/routes.rb]: Ready

**Pull request merged**